clients/src/main/resources/common/message/DescribeConfigsRequest.json

```diff
@@ -32,6 +32,8 @@
         "about": "The configuration keys to list, or null to list all configuration keys." }
     ]},
     { "name": "IncludeSynoyms", "type": "bool", "versions": "1+", "default": "false", "ignorable": false,
```

Comment:
Unrelated to this PR, but since you are updating this file anyway, can you fix the typo in `IncludeSynonyms`?